docs/headless-cms/03-03-metadata.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Metadata
+type: Details
+---
+
+Apart from content, each markdown document displayed in the Kyma Console requires metadata in a specific format called [Front Matter](https://forestry.io/docs/editing/front-matter/).
+
+## Metadata structure
+
+When you create a document, define its `title` and `type`. Place the metadata at the top of your `.md` file, and separate it with three dashes. The following example illustrates the required structure:
+
+```
+---
+title: {Document title}
+type: {Document type}
+---
+```
+
+- The `title` metadata defines the title of your document. 
+- The `type` metadata groups single documents together. Multiple documents that use the same `type` generate a grouping. For example, if you have multiple tutorials, you can group them below a navigation node called **Tutorials**.
+
+>**NOTE:** If there is only one document of a certain type, remove the `type` metadata, so that the document displays well in the UIs.
+
+## Metadata display
+
+In the Docs UI, which is the view displayed once you click on the question mark icon (![](./assets/docs-ui-question-mark.png)) in the top-right corner of the Console, the metadata help create the left-side navigation structure. The Docs UI displays documents grouped under a common `type` in alphanumeric order as per files names. The following example shows three documents, their metadata, and corresponding places in the left-side navigation.
```

Comment:
(![](./assets/docs-ui-question-mark.png)) looks strage, remove brackets around the icon picture.